### ISSUE TITLE
Exception traceback logs

### ISSUE THREAD

**Issue by AliSayyah** [indietyp/django-automated-logging#9]
thank you for your app
but I have a problem
it seems that it doesn't save traceback logs after exceptions occurred or for 500 responses.
is there a way to set this?

**Comment by indietyp** [comment]
The default for the application is to ignore all GET requests, you can change that behavior in the settings dictionary.

**Comment by AliSayyah** [comment]
yes, I understand that. but my problem is about POST requests. Django-auto-logger saves failed requests in the database, but traceback logs and exceptions are not saved there. So I can't really use it to debug the problem that occurred.

**Comment by indietyp** [comment]
I am sorry. The database logs are intended to be lightweight in that regard and to be used in production.

I would be open to adding a configuration setting to allow saving tracebacks.
